### PR TITLE
feat: improve config for Remotec zxt-310

### PR DESCRIPTION
**Pull request by blhoward2**
Reuqested via discord. Corrects mismapped device files. Adds parameters.

### CODE REVIEW

File: packages/config/config/devices/0x5254/bw8510cn.json [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-{
-	"manufacturer": "Remotec",
-	"manufacturerId": "0x5254",
-	"label": "BW8510CN",
-	"description": "ZRC-90CN",
-	"devices": [
-		{
-			"productType": "0x000b",
-			"productId": "0x8510",
-			"zwaveAllianceId": 1585
```
Comment (AlCalzone) [inline packages/config/config/devices/0x5254/bw8510cn.json:12]: So this one doesn't exist?

File: packages/config/config/devices/0x5254/zxt-310.json [hunk 2]
```diff
@@ -9,44 +9,218 @@
...
+				},
+				{
+					"label": "IR Port 5",
+					"value": 5
+				},
+				{
+					"label": "IR Port 6",
+					"value": 6
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x5254/zxt-310.json:165]: Does this need options?

File: packages/config/config/devices/0x5254/zxt-310.json [hunk 3]
```diff
@@ -9,44 +9,218 @@
...
+				},
+				{
+					"label": "Endpoint 5",
+					"value": 5
+				},
+				{
+					"label": "Endpoint 6",
+					"value": 6
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x5254/zxt-310.json:223]: Same here - does this need options?

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x5254/bw8510cn.json (hunk 1 above)
Comment (blhoward2) [reply packages/config/config/devices/0x5254/bw8510cn.json:12]: I missed pushing the file I added it to. It's in now.

File: packages/config/config/devices/0x5254/zxt-310.json (hunk 2 above)
Comment (blhoward2) [reply packages/config/config/devices/0x5254/zxt-310.json:165]: I debated that. I'm fine either way but it seems like something you'd pick in a drop down?

File: packages/config/config/devices/0x5254/zxt-310.json (hunk 3 above)
Comment (blhoward2) [reply packages/config/config/devices/0x5254/zxt-310.json:223]: Same

File: packages/config/config/devices/0x5254/zxt-310.json (hunk 2 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x5254/zxt-310.json:165]: Ok for me

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**